exchanges/btcmarkets/btcmarkets.go

```diff
@@ -370,7 +544,7 @@ func (b *BTCMarkets) SendHTTPRequest(path string, result interface{}) error {
 		path,
 		nil,
 		nil,
-		result,
+		&result,
```

Comment:
Like SendAuth, you only need to use a reference if you haven't already specified it in the calling func